gps_imu_node:
  ros__parameters:
    host: 192.168.1.172
    port: 60570
    gps_topic: /gps
    imu_topic: /imu
    debug: True